packages/loopring_v3/contracts/impl/owners/LoopringV3Owner.sol

```diff
@@ -19,19 +19,137 @@ pragma solidity ^0.5.11;
 import "./DelayedOwner.sol";
 import "../../iface/ILoopringV3.sol";
 
+import "./ChainlinkTokenPriceOracle.sol";
+
 
 /// @title LoopringV3Owner
 /// @author Brecht Devos - <brecht@loopring.org>
 contract LoopringV3Owner is DelayedOwner
 {
+    uint public constant NUM_MOVING_AVERAGE_DATA_POINTS = 7;
+    uint public constant MOVING_AVERAGE_TIME_PERIOD = 1 days;
```

Comment:
Brecht, can we have a separate contract that provides the usdToLRC function using the 7-day moving average so other contracts (such as DowntimeCostCalculator) can also use?